sql/src/test/java/io/crate/expression/scalar/timestamp/AtTimezoneSyntaxFunctionTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.expression.scalar.timestamp;
+
+import io.crate.expression.scalar.AbstractScalarFunctionsTest;
+import io.crate.expression.symbol.Literal;
+import io.crate.types.DataTypes;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.time.format.DateTimeParseException;
+
+import static java.lang.String.format;
+
+public class AtTimezoneSyntaxFunctionTest extends AbstractScalarFunctionsTest {
```

Comment:
I think we could instead write a test for the parser that verifies that we generate a `FunctionCall` and then only have a single test here, to verify the integration between `AT TIME ZONE` and the  `timezone` scalar. No need to test all edge cases again.

Reply:
done